core/domain/event_services_test.py

```diff
@@ -290,6 +532,13 @@ def setUp(self):
             self.save_new_valid_exploration('exp_id', self.admin_id))
 
     def test_average_ratings_of_users_exps_are_calculated_correctly(self):
+
+        # If user id is not present in UserStatsModel.
+        event_services.handle_exploration_rating('exp_id', 5, None)
+        admin_average_ratings = (
```

Comment:
You'll need a pre-check for this, I think (to make sure it wasn't 5 before).

Reply:
I already did that. Refer to the code just above it.
```
admin_average_ratings = (
            user_services.get_dashboard_stats(self.admin_id)['average_ratings'])
        self.assertEqual(admin_average_ratings, None)
```
 Although, I have now changed `self.assertIsNone(admin_average_ratings) `to  `self.assertEqual(admin_average_ratings, None)` just to match the pattern used everywhere else.

Reply:
Ok! I have added that check.